akka-docs/src/main/paradox/typed/distributed-data.md

```diff
@@ -270,8 +270,8 @@ better safety for small clusters.
 #### Consistency and response types
 
 When using `ReadLocal`, you will never receive a `GetFailure` response, since the local replica is always available to
-local readers. `WriteLocal` however may still reply with `UpdateFailure` messages, in the event that the `modify` function
-threw an exception, or, if using @ref:[durable storage](#durable-storage), if storing failed.
+local readers. `WriteLocal` however may still reply with `UpdateFailure` messages when the `modify` function
+throws an exception, or if storing failed when using @ref:[durable storage](#durable-storage).
```

Comment:
Shouldn't it either be "if ..., or if ..." or both "when ..., or when", now it is "when ..., or if ..."?

Reply:
Good catch, changed to "if ..., or if ..."